exercises/high-scores/canonical-data.json

```diff
@@ -0,0 +1,99 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the list of scores",
+      "property": "scores",
+      "input": {
+        "scores": [3, 5, 2, 7]
+      },
+      "expected": [3, 5, 2, 7]
+    },
+    {
+      "description": "Returns the last score",
+      "property": "latest",
+      "input": {
+        "scores": [10, 0, 9, 3]
+      },
+      "expected": 3
+    },
+    {
+      "description": "Returns the highest score",
+      "property": "highest",
+      "input": {
+        "scores": [4, 10, 7]
+      },
+      "expected": 10
+    },
+    {
+      "description": "Returns the top 3 scores",
+      "cases": [
+        {
+          "description": "Returns personal bests when last is less",
+          "property": "top",
+          "input": {
+            "scores": [5, 3, 1]
+          },
+          "expected": [5, 3, 1]
+        },
+        {
+          "description": "Returns personal bests including zero",
+          "property": "top",
+          "input": {
+            "scores": [2, 0, 3]
+          },
+          "expected": [3, 2, 0]
+        },
```

Comment:
The goal here wasn't exactly the 0. It was the first (in the TDD sense) where the order was not the same as the initial order.

But maybe we just remove "Returns personal bests when last is less"? We can also change 0 to be something else.